controllers/dashboardController.js

```diff
@@ -0,0 +1,63 @@
+
+const jwt = require("jsonwebtoken")
+
+const config = require("../config")
+const conferences = require("../lib/conferences")
+const { decrypt } = require("../lib/crypto")
+
+module.exports.get = async (req, res) => {
+    const token = req.params.token
+
+    if (!token) {
+        res.redirect('/')
+    }
+
+    try {
+        const roomNumber = jwt.verify(decrypt(token), config.SECRET).roomNumber
+        const participantIds = await conferences.getParticipants(config.OVH_ROOM_PHONE_NUMBER, roomNumber)
+        const participants = await Promise.all(participantIds.map(id => conferences.getParticipant(config.OVH_ROOM_PHONE_NUMBER, roomNumber, id)))
+        return res.render("dashboard", {
+            participants: participants.map(participant => {
+                const arrivalDateTime = participant.arrivalDateTime
+                const date = new Date(arrivalDateTime)
+                return {
+                    ...participant,
+                    arrivalTime: date.getHours() + ":" + date.getMinutes() + ":" + date.getSeconds(),
+                    callerNumber: participant.callerNumber.slice(0, 4) + "XXXX" + participant.callerNumber.slice(-4)
+                }
+            }),
+            phoneNumber: config.OVH_ROOM_PHONE_NUMBER,
+            roomNumber,
+            token,
+            lastUpdate: new Date()
+        })
+    } catch (err) {
+        console.log(`Impossible de recuperer la room : ${err}`)
+        req.flash(`Ce lien n'est plus valide : ${err}`)
```

Comment:
je sais pas si c'est comprehensible d'afficher ${err} à l'utilisateur... Je ferais plutot : 
```suggestion
        req.flash(`Ce lien n'est plus valide. Vous pouvez recréer une conférence.`)
```